fiasco/ions.py

```diff
@@ -94,7 +99,7 @@ def __repr__(self):
 HDF5 Database: {self.hdf5_dbase_root}
 Using Datasets:
   ioneq: {self._dset_names['ioneq_filename']}
-  abundance: {self._dset_names['abundance_filename']}
+  abundance: {self._dset_names['abundance']}
```

Comment:
Alternatively, this will return the dataset name if it exists and if it doesn't, just returns the custom value.

```suggestion
  abundance: {self._dset_names.get('abundance', self.abundance)}
```

Reply:
I've changed this as suggested, but I'm not sure I understand this one.  Please verify!

Reply:
Yep, looks good. By calling `get` as opposed to just accessing that entry in the dict directly, you allow for the possibility that the `'abundance'` key is not in `_dset_names`. If it's not, it falls back to the value returned by `self.abundance`.